tests/python/unittest/test_arith_solve_linear_system.py

```diff
@@ -0,0 +1,213 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import random
+import numpy as np
+import tvm
+from tvm import te, arith, ir, tir
+
+
+def run_expr(expr, vranges):
```

Comment:
Some comment like "Evaluate expr for every value of free variables given by vranges and return the tensor of results.".
Also I think these functions are quite useful and need to be moved somewhere like tvm.testing (but probably not in this PR, so may be add a TODO).